pkg/ingester/ingester.go

```diff
@@ -103,7 +103,9 @@ type Config struct {
 	LifecyclerConfig ring.LifecyclerConfig `yaml:"lifecycler,omitempty"`
 
 	// Config for transferring chunks. Zero or negative = no retries.
-	MaxTransferRetries int `yaml:"max_transfer_retries,omitempty"`
+	MaxTransferRetries        int `yaml:"max_transfer_retries,omitempty"`
+	MinTransferRetriesBackOff int `yaml:"min_tranfer_retries_backoff,omitempty"`
```

Comment:
Few things:
1. There's a typo in the yaml: `tranfer` > `transfer`. The same issue applies to the next line.
2. I would suggest to name it `transfer_(max|min)_backoff_period` / `-ingester.transfer-(max|min)-backoff-period` to keep consistency with other backoff settings in cortex
3. The variable name should be specular to the new parameter naming (and `BackOff` > `Backoff`)